gtests/net/mptcp/fastclose/receive_fastclose_with_ack_multi_v4.pkt

```diff
@@ -15,9 +15,8 @@
 +0    accept(3, ..., ...) = 4
 
 // add_addr
-+0.0    >                                .  1:1(0)        ack 1                <add_address addr[saddr1] hmac=auto, dss dack4=1                    dll=0    nocs>
-// TODO: send echo bit
-// +0.0 <                                .  1:1(0)        ack 1     win 256    <add_address addr[saddr1],           dss dack8=1                    dll=0    nocs>
++0.0    >                                .  1:1(0)        ack 1                <add_address address_id=1 addr[saddr1] hmac=auto, dss dack4=1                    dll=0    nocs>
++0.0    <                                .  1:1(0)        ack 1     win 256    <add_address address_id=1 addr[saddr1] addr_echo, dss dack8=1                    dll=0    nocs>
```

Comment:
I don't know if we should align the dss here, it starts to be quite wide.

Reply:
@matttbe 
```
[dcaratti@bottarga packetdrill]$ stty size
55 254
```
and the DSS is at column 130. I'm still in favour of putting scripts "in column", but that should only be done for data that evolve differently withing individual TCP subflows. DSN and DACK numbers (more generally, MPTCP options) are "in common" so they should still be readable also if we align them to a single column. That might not fix readability for all scripts, but it should help most cases. WDYT?

Reply:
it sounds good to me

(I hope I understand that properly :) )